tests/indices/create_or_update_alias.yaml

```diff
@@ -0,0 +1,42 @@
+$schema: ../../json_schemas/test_story.schema.yaml
```

Comment:
We already have `alias.yaml` that has a lot of similar tests. Either add one more test for `is_hidden` there, or create a folder called `tests/indices/alias/is_hidden.yaml` and include only functionality related to this new is_hidden into it. I think the former seems simpler.

You also need to add `version: '>= 2.16'` into the test so it doesn't run with older OpenSearch (I expect it to fail).